grpc-testtool/proto/merkle/merkle.proto

```diff
@@ -5,48 +5,48 @@ package merkle;
 import "google/protobuf/empty.proto";
 
 service Merkle {
-    rpc NewProposal(NewProposalRequest) returns (NewProposalResponse);
-    rpc ProposalCommit(ProposalCommitRequest) returns (ProposalCommitResponse);
+  rpc NewProposal(NewProposalRequest) returns (NewProposalResponse);
+  rpc ProposalCommit(ProposalCommitRequest) returns (google.protobuf.Empty);
```

Comment:
We now use status codes to convey the reason for an error instead of returning it as a field in the response